### ISSUE TITLE
Derive CES network tax

### ISSUE THREAD

**Issue by irudik** [irudik/climate-networks#1]
- CES materials and CES utility to at least be able to use the CPT strategy for recovering productivities and distortions
- If temperature hits factor productivity, then we should be able to derive a nice network formulation for the optimal carbon tax
- Suppose the use of intermediate k in production if j is what causes emissions and warming, k produces emissions at some constant rate (ignore abatement/altering carbon intensity for now)
- This will raise the socially efficient price for k by how much 1 unit of k reduces output across all industries
- This is the efficient tax, basically doutput/dshock
- The review article shows how to express dlog(output)/dshock in terms of network structure for Cobb-Douglas: log(y) = L*shock + constant -> dlog(y)/dshock_j = sum(L_{ij}, i) where the Ls measure the importance of j in supplying i
- The last bullet point shows the network structure: when j gets hit by bad temperature caused by emissions by using k, it feeds into every other industry i
- Appendix A does it for CES

**Comment by weiliangt** [comment]
Ok I'll read CPT and the review article in detail in a few weeks time when I work on the network stuff. Will think about the points above and try to derive the network tax then

**Comment by irudik** [comment]
Yeah no worries. I'm just getting starting laying out the derivations for everything the next few days before I have to get back to some other projects.

**Comment by irudik** [comment]
Introduce distortion similar to CPT where there's a wedge between the true price of an intermediate and the actual cost of using it. This should feed into the tax expression. We can then estimate the distortions later and compute welfare from second-best tax versus blindly setting tax = MD.

**Comment by weiliangt** [comment]
Looked through your derivation. I can finish it up once I work on it in a week or two. Some thoughts:
- Probably the key thing we need is to have the distortion \kappa and the Leontief inverse in the full optimal tax formula to show the intuition theoretically. Then we can have a proposition on the comparative statics. 
- We probably should emphasize that our Leontief inverse is endogenous since we have endogenous Dolmar weights with CES. Might also be interesting to see how temperature affects the endogenous Dolmar weights, so we can both theoretically and quantitatively assess the role of an endogenous production structure in determining climate impacts -- some parallels to the adaptation idea in our contagion paper. Also we can explore the interactions between endogenous production structure and existing distortions
